app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+import re
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source_path, destination_path = command.split()
+    except ValueError:
+        print("Wrong command format")
+    else:
+        if command == "mv" and source_path != destination_path:
+            if not os.path.isfile(source_path):
+                raise ValueError(f"File '{source_path}' doesn't exist")
```

Comment:
You can simplify your code but not checking source path here but instead wrap the block of code where you read/write the file in try except - this will allow to process all file-related error including wrong path